.github/workflows/start-jedi-ci.yaml

```diff
@@ -0,0 +1,51 @@
+name: start-jedi-ci
+
+on:
+  pull_request:
+    branches:
+     - 'master'
```

Comment:
We don't use master or main branches.

Reply:
Just casting a broad net here. This clause is the same on all repos which also makes global changes (which require scripted github interactions) a little easier.